gm4_chairs/data/chairs/functions/kill.mcfunction

```diff
@@ -0,0 +1,7 @@
+# @s = @e[tag=gm4_chairs] and NOT inside stair blocks
+#execute at @s align xyz run tp @e[nbt={RootVehicle:{Entity:{id:"minecraft:pig"}}},dx=0,dz=0,dy=2] ~ ~.4 ~
+tp @s ~ ~.5 ~
+execute at @s run summon minecraft:area_effect_cloud ~ ~ ~ {Duration:100,Tags:["gm4_chairs_temp"]}

```

Comment:
I don't see why a marker needs to be summoned. `at @s` is `positioned ~ ~.5 ~` atm